clientv3/watch.go

```diff
@@ -25,6 +25,12 @@ import (
 	"google.golang.org/grpc"
 )
 
+const (
+	DeleteEventType = storagepb.DELETE
+	ExpireEventType = storagepb.EXPIRE
+	PutEventType    = storagepb.PUT
+)
+
 type WatchChan <-chan WatchResponse
```

Comment:
There is one problem I had on this.
For copying pb.WatchResponse.Events to WatchResponse.Events, like [here](https://github.com/coreos/etcd/blob/b0cc0e443c163f36f29d078dd855ca9c7b68e612/clientv3/watch.go#L358), it will complain error.